homeassistant/components/icloud/account.py

```diff
@@ -354,6 +354,11 @@ def fetch_interval(self) -> int:
         """Return the account fetch interval."""
         return self._fetch_interval
 
+    @property
+    def gps_accuracy_threshold(self) -> int:
+        """Return configured GPS Accuracy Threshold."""
+        return self._gps_accuracy_threshold
+
```

Comment:
I'm not very familiar with Python but I'm really trying to get this stuff right, so bear with me. 

I just read online that it might not be good practice to use a getter/property in this context.
Should the variable `self._gps_accuracy_threshold` just be made public? So that `self.gps_accuracy_threshold` can be read in the device class?

Would appreciate some feedback so that the code-quality doesn't suffer :)